scenarios/kubernetes_e2e.py

```diff
@@ -55,8 +55,7 @@ def kubekins(tag):
     """Return full path to kubekins-e2e:tag."""
     return 'gcr.io/k8s-testimages/kubekins-e2e:%s' % tag
 
-# pylint: disable=too-few-public-methods
-class BoskosProject(object):
+class BoskosProject(object): # pylint: disable=too-few-public-methods
```

Comment:
Since this is inline the scope of this is limited to the class definition. You do not need to enable it again after leaving this scope.

Aka it is enabled at top level, class BoskosProject inherits top-level lint checks and the comment disables too-few-public-methods, class LocalMode also inherits top-level lint checks, which excludes anything disabled for other class scopes.